sklearn/metrics/tests/test_common.py

```diff
@@ -1116,3 +1116,16 @@ def test_no_averaging_labels():
             score_labels = metric(y_true, y_pred, labels=labels, average=None)
             score = metric(y_true, y_pred, average=None)
             assert_array_equal(score_labels, score[inverse_labels])
+
+
+@ignore_warnings
+def test_all_true_pos_label():
```

Comment:
Thanks for this.

Should we be explicitly passing pos_label=1?

And I'm not sure about what we're saying about metrics where the second arg is a score or probability here...

Reply:
Changed it to explicitly pass that.
I thought the test would be like the equivalent of `check_classifiers_one_label` but for those metrics.